### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by charlynet7** [angular-ui/AngularJS-Atom#31]
Hi,
I'm getting this warning.
I'm using the version 1.0, I don't know why it doesn't allow me to install the v.2.0.
I'm using the last version of Atom ( v.0.197.0 ).

Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (C:\Users\<user>\AppData\Local\atom\app-0.197.0\resources\app.asar\src\package.js:586:9)
<unknown> (C:\Users\<user>\AppData\Local\atom\app-0.197.0\resources\app.asar\src\package.js:188:43)
Package.measure (C:\Users\<user>\AppData\Local\atom\app-0.197.0\resources\app.asar\src\package.js:167:15)
Package.load (C:\Users\<user>\AppData\Local\atom\app-0.197.0\resources\app.asar\src\package.js:181:12)
```

**Comment by outsideris** [comment]
Deprecated APIs is fixed.
But, I don't know why users use 0.1.x instead 0.2.0 and I'm not sure it is caused by Atom or this package.

**Comment by charlynet7** [comment]
Hi,
This is with Atom 0.199.0
This is a very fresh new install (I deleted all the .atom folder on the users folders, so all the packages data with it)

![image](https://cloud.githubusercontent.com/assets/4017756/7663448/510f5346-fb1f-11e4-8de2-3a08091af0bb.png)

**Comment by joshuajabbour** [comment]
Not sure why automatic upgrade isn't working, but it can be upgraded manually with `apm install angularjs@0.2.0`.

**Comment by outsideris** [comment]
I expect It fixed in #39 . I will publish new version.
